gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageClientImpl.java

```diff
@@ -75,6 +77,7 @@ public class GoogleCloudStorageClientImpl extends ForwardingGoogleCloudStorage {
       @Nullable Credentials credentials,
       @Nullable HttpTransport httpTransport,
       @Nullable HttpRequestInitializer httpRequestInitializer,
+      @Nullable List<ClientInterceptor> gRPCInterceptors,
```

Comment:
Why not use an empty list, instead of nullable? It will make the code a bit more safe.